packages/reports/addon/consumers/request/rollup.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import { inject as service } from '@ember/service';
+import ActionConsumer from 'navi-core/consumers/action-consumer';
+import Route from '@ember/routing/route';
+import RequestActionDispatcher, { RequestActions } from 'navi-reports/services/request-action-dispatcher';
+import ColumnFragment from 'navi-core/addon/models/bard-request-v2/fragments/column';
+import ReportModel from 'navi-core/models/report';
```

Comment:
Check for other `/addon/` in imports

```suggestion
import RequestActionDispatcher, { RequestActions } from 'navi-reports/services/request-action-dispatcher';
import type Route from '@ember/routing/route';
import type ColumnFragment from 'navi-core/models/bard-request-v2/fragments/column';
import type ReportModel from 'navi-core/models/report';
```